control-plane-operator/controllers/hostedcontrolplane/kas/ibmcloud_kms.go

```diff
@@ -152,6 +160,19 @@ func buildVolumeIBMCloudKMSKP(v *corev1.Volume) {
 	v.Secret.Optional = &optionalMount
 }
 
+func buildVolumeIBMCloudKMSProjectedToken(v *corev1.Volume) {
+	v.Projected = &corev1.ProjectedVolumeSource{
```

Comment:
I'm curious is this projecting a token coming from the management side? Can you provide more context on the PR desc on how this would work?

Reply:
Yea! I can get you some details